db.js

```diff
@@ -111,7 +111,12 @@ export const saveWFS = async (wfsUrl, user, password) => {
 
 export const deleteWFS = wfs => {};
 
-export const syncWFS = wfs => {};
+export const syncWFS = wfs => {
+  realm.objects('WFS').filtered(`url == "${wfs.url}"`).forEach(wfs => {
```

Comment:
This line isn't necessary, the wfs parameter into the function is the realm object